Assets/MixedRealityToolkit/Definitions/InputSystem/MixedRealityRaycastHit.cs

```diff
@@ -35,7 +35,7 @@ public MixedRealityRaycastHit(bool raycastValid, RaycastHit hitInfo)
                 triangleIndex = hitInfo.triangleIndex;
                 textureCoord = hitInfo.textureCoord;
                 textureCoord2 = hitInfo.textureCoord2;
-                transform = hitInfo.transform;
+                transform = hitInfo.collider.transform; // we're using collider transform because hitInfo.transform might point us to a different gameobject in case there's a rigidbody in the hierarchy
```

Comment:
>transform = hitInfo.collider.transform; //  [](start = 16, length = 43)

This was something that was desired by one of the app teams in mrw a while back, and we had added a profile configuration property to address this - focusIndividualCompoundCollider so that apps could choose which transform they prefer to pick - 
Check out commit - [https://github.com/microsoft/MixedRealityToolkit-Unity/commit/68994887db97a081c0483c2a79db4503fe5b570b#diff-7b0c1b5a3f72bb7ada11d8478aa6fbbb](https://github.com/microsoft/MixedRealityToolkit-Unity/commit/68994887db97a081c0483c2a79db4503fe5b570b#diff-7b0c1b5a3f72bb7ada11d8478aa6fbbb)

This change feels redundant to that commit.

Reply:
I wasn't aware of that. Thanks @ritijain!